daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -160,6 +159,20 @@ def DistroSpecific(g):
   g.command(['update-grub2'])
 
 
+def remove_azure_agents(g):
+  try:
+    g.command(['apt-get', 'remove', '-y', '-f', 'walinuxagent'])
+  except Exception as e:
+    logging.debug(str(e))
+    logging.warn('Could not uninstall Azure agent. Continuing anyway.')
+
+  try:
+    g.command(['apt-get', 'remove', '-y', '-f', 'waagent'])
+  except Exception as e:
+    logging.debug(str(e))
+    logging.warn('Could not uninstall Azure agent. Continuing anyway.')
+
+
```

Comment:
What's the relationship between waagent and walinuxagent?
Is it expected that both are present at the same time? 

If only one can be present, showing the warning message for the other one might be confusing. Also, showing the same warning twice.

Can we differentiate between the two in warning messages?

Reply:
I made an Ubuntu 18.04 instance in Azure, and it had walinuxagent but not waagent. I posted a new commit that differentiates the error messages. 

I'm not aware of any condition that requires one or the other to be present. This just makes it flexible in case one is present and the other is not.